src/style.css

```diff
@@ -0,0 +1,19 @@
+.red {
+  background-color: red;
+}
+
+.green {
+  background-color: green;
+}
+
+.blue {
+  background-color: blue;
+}
+
+.noMargin {
+  margin: 0;
+}
+
+.high100 {
```

Comment:
i think it's bad idea to make this class, better to create common class for all divs and get them height 100px